scripts/conda-activate-env.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if conda env list | grep -q "^firesim"; then
+    conda activate firesim
+else
+    conda activate chipyard
+fi
```

Comment:
When would there be a `firesim` env? Why not just tell the user to always `conda activate chiypard`?

Reply:
FireSim by default calls it's env `firesim`. So if we want to support Chipyard in the FireSim setup (on AWS F1) then we should switch between them.

Reply:
The docs say
>  Warning:: When running on an Amazon Web Services EC2 FPGA-development instance (for FireSim), FireSim includes a similar machine setup script that will install all of the aforementioned dependencies (and some additional ones). Skip this section.

This instructs users running Chipyard in FireSim to skip running this script, since I believe the fsim docs should contain instructions for activating the firesim env. In that case, this script doesn't need to exist, and the docs can just say `conda activate chipyard`

For my own understanding, I believe the flow is:
1. If FireSim, follow firesim docs, then skip to 5
2. install conda
3. create chipyard conda env
4. activate conda env
5. init submodules

Reply:
This was re-worded in the past. Better now?